src/app/webcontainer/WebApp.qml

```diff
@@ -370,6 +370,12 @@ Common.BrowserView {
                           })
             }
 
+            onStatusChanged: {
+                if (status != Loader.Ready) {
+                    webapp.currentWebview.zoomController.reset()
+                }
+            }
+
```

Comment:
note: if https://github.com/ubports/morph-browser/pull/255 is merged, this can be removed again (the refactored zoom controller handles the change itself)